akka-actor/src/main/scala/akka/actor/ActorCell.scala

```diff
@@ -680,25 +684,16 @@ private[akka] class ActorCell(
     case _                               =>
   }
 
-  final protected def clearActorCellFields(cell: ActorCell): Unit = {
-    cell.unstashAll()
-    if (!Reflect.lookupAndSetField(classOf[ActorCell], cell, "props", ActorCell.terminatedProps))
-      throw new IllegalArgumentException("ActorCell has no props field")
-  }
-
+  @silent
   final protected def clearActorFields(actorInstance: Actor, recreate: Boolean): Unit = {
```

Comment:
`actorInstance ` and `recreate` not used and can be removed?

Reply:
Done by now.